custom/enikshay/const.py

```diff
@@ -60,6 +60,11 @@
 
 # BETS event names
 VOUCHER_EVENT_ID = "VOUCHER_EVENT_ID"  # TODO: Get this from BETS
+TREATMENT_180_EVENT = "TREATMENT_180_EVENT"  # TODO: Get this from BETS
```

Comment:
we have all of these. in an email thread called "(eNikshay - BETS integration) Meeting notes: 30 March".
Email from abhijeet on on April 21